azure-quantum/environment-cirq-beta.yml

```diff
@@ -17,8 +17,8 @@ dependencies:
     - azure-core==1.19.1 # see https://github.com/microsoft/qdk-python/issues/220 
     - azure-devtools>=1.2.0
     - azure-storage-blob==12.11.0 # see https://github.com/microsoft/qdk-python/issues/220
-    - --pre cirq-core>=0.13.1
-    - cirq-ionq>=0.13.1
+    - --pre cirq-core>=0.15.0
```

Comment:
the point of this environment is to test Cirq with their beta version, so this should be pointing to 0.16 (it should point to 0.15 earlier, which would have allowed us to fix the problem before).

Reply:
Interesting that we had both in 0.13.1.
All right, let me update this one.